backend/src/main/java/org/eclipse/tractusx/puris/backend/masterdata/domain/model/Partner.java

```diff
@@ -39,6 +46,9 @@ public class Partner {
     @Id
     @GeneratedValue
     private UUID uuid;
+    /**
+     * The full name of the partner.
+     */
     private String name;
     private String edcUrl;
```

Comment:
please add also text to the others. the edcUrl e.g. is the protocol path of the partners edc. now its ids, soon dsp

Reply:
I have extended the commenting there. Since the Partner class is subject to some refactoring at the moment (Refactor/partner sitebpns #27), I´d be happy to add further commenting, as soon as this other issue has been (hopefully) merged.

Reply:
Good idea.